src/components/Achieve.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import AchieveBottom from './AchieveBottom'
+import './AchieveCss.css'
+import AchieveTop from './AchieveTop'
+
+const Achieve = () => {
+  return (
+    <section className="achieve-container">
+      <AchieveTop />
```

Comment:
I think this is a really good way to structure and organize a big page with more code!